src/hooks/useKeyPicker.ts

```diff
@@ -0,0 +1,86 @@
+import { useCallback, useEffect, useState } from 'react';
+import { DigiKey } from '../types/DigiKey';
+import { MAX_PRONGS } from '../constants';
+import { rotateKey } from '../utils/rotateKey';
+
+const quickKeys: DigiKey[] = [
+  [1],
+  [1, 3],
+  [1, 3, 31],
+  [1, 5, 29],
+  [31, 27, 3, 7],
+  [31, 25, 5, 11],
```

Comment:
What are these values? Could we move them to a constant with a name like `DEFAULT_KEYS`?